code/drasil-docLang/lib/Drasil/Sections/SpecificSystemDescription.hs

```diff
@@ -280,8 +297,9 @@ helperCI a c = over defn (\x -> foldlSent_ [x, refby $ helperRefs a c]) a
     refby sent   = sParen $ S "RefBy:" +:+. sent
 
 -- | Section stubs for implicit referencing of different models and definitions.
-tmStub, ddStub, imStub, pdStub :: Section
+tmStub, ddStub, imStub, pdStub, gdStub :: Section
```

Comment:
I would reorder this to be consistent with the order of their definitions.
```suggestion
tmStub, ddStub, gdStub, imStub, pdStub :: Section
```

Reply:
Ok, I will do that right now!